run-qa-checks

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -e
+
+# will be 1 by default when run through github actions
+CI=${CI:-0}
+
+if [ "$CI" -eq 1 ]; then
+	# copy files in github actions so that they can be formatted
+	cp openwisp-config/files/openwisp.agent openwisp_agent
+	cp openwisp-config/files/openwisp.init openwisp_init
+else
+	# check shell scripts formatting
+	sh_files=$(shfmt -f .)
+	# shellcheck disable=SC2086
+	shellcheck $sh_files \
+		openwisp-config/files/openwisp.agent \
+		openwisp-config/files/openwisp.init
+
+	shfmt -d .
+	shfmt -d openwisp-config/files/openwisp.agent
+	shfmt -d openwisp-config/files/openwisp.init
+fi
```

Comment:
`shfmt` is either available in snap store or through docker image. For CI build I preferred docker image via github `luizm/action-sh-checker@master`.
However, only those executable files are treated as shell files that have either `.sh` extension or no extension.
So, I copy `openwisp agent` and `openwisp daemon` in case of CI build without extension to test it via docker.

In local case, I tested both agent and daemon one by one, as it seems best we can done while working on `openwrt openwisp monitoring`

Reply:
In these cases, always add comments please, because it's not clear by reading this file only what is going on, it seems that the shell checks are being omitted in the CI.

Reply:
https://github.com/openwisp/openwisp-config/commit/0e94c23d66989296c58dc2aba9c51ab562b4167f